lib/controllers/mongo.js

```diff
@@ -23,6 +24,24 @@ Mongo.prototype.create = function (name, creatorId, regionData, callback) {
 }
 
 // -----------------------------------------------------------------------------
+Mongo.prototype.clone = function (db, hosts, callback) {
+  var api = userConfig.data.api_uri.split('://')
+
+  var Client = apiUtil({
+    protocol: api[0],
+    hostname: api[1],
+    port: api[0] === 'https' ? 443 : 80,
```

Comment:
going off the docs in the `api-util` repo, but I can remove it if it's not needed